cadc-access-control-server/src/main/java/ca/nrc/cadc/ac/server/oidc/OIDCUtil.java

```diff
@@ -415,5 +412,15 @@ static String getClaimDescriptionString(List<Claim> claims) {
         }
         return sb.toString();
     }
+
+    public static String getEncodedKID() {
+        return Base64.getUrlEncoder().encodeToString(KID_CLAIM_VALUE.getBytes());
```

Comment:
I wonder if this should use a one-way encoder so that clients can't read the contents... Did you see any recommendations on that?

Reply:
As discussed in slack - the KID_CLAIM_VALUE is going to be a simple string unrelated to the file name for the RSA priv/pub keys, not encoded.